common/app/views/fragments/atoms/storyquestions.scala.html

```diff
@@ -39,12 +33,21 @@ <h2 class="user__question-title">
 
                 <button type="submit" class="js-storyquestion-vote-button storyquestion-vote-button button button--primary">Vote</button>
 
-                <button type="submit" class="is-hidden js-storyquestion-email-question-button storyquestion-email-question-button button button--primary">Email me</button>
-
                 @storyquestions.atom.labels.find(_.length == 4).map { listId =>
+
+                    <div class="is-hidden js-storyquestion-email-question storyquestion-email-question">We're working on the answer. Would you like to receive an email when we answer the most popular question?</div>
+
+                    <div class="is-hidden js-storyquestion-email-enter storyquestion-email-enter">Enter your email</div>
```

Comment:
And this one is a label, so `<label for="#email-input-@listId" class="is-hidden ...">Enter ...</label>`